commands/cli.go

```diff
@@ -0,0 +1,63 @@
+package commands
+
+import (
+	"strings"
+
+	"github.com/gruntwork-io/aws-nuke/aws"
+	"github.com/gruntwork-io/aws-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/gruntwork-io/gruntwork-cli/shell"
+	"github.com/urfave/cli"
+)
+
+// CreateCli - Create the CLI app with all commands, flags, and usage text configured.
+func CreateCli(version string) *cli.App {
+	app := cli.NewApp()
+
+	app.Name = "aws-nuke"
+	app.HelpName = app.Name
+	app.Author = "Gruntwork <www.gruntwork.io>"
+	app.Version = version
+	app.Usage = "A CLI tool to cleanup AWS resources (EC2). THIS TOOL WILL COMPLETELY REMOVE ALL RESOURCES AND ITS EFFECTS ARE IRREVERSIBLE!!!"
+	app.Action = errors.WithPanicHandling(awsNuke)
+
+	return app
+}
+
+// Nuke it all!!!
+func awsNuke(c *cli.Context) error {
+	logging.Logger.Infoln("Retrieving all active AWS resources")
+
+	resources, err := aws.GetAllResources()
+
+	if err != nil {
+		return errors.WithStackTrace(err)
+	}
+
+	if len(resources) == 0 {
+		logging.Logger.Infoln("Nothing to nuke, you're all good!")
+		return nil
+	}
+
+	for resource, resourceMap := range resources {
+		for region, instances := range resourceMap {
+			for _, instance := range instances {
+				logging.Logger.Infof("%s-%s-%s", resource, *instance, region)
+			}
+		}
+	}
+
+	prompt := "\nAre you sure you want to nuke all listed resources? Enter 'nuke' to confirm: "
```

Comment:
I would still add a log line before this with a big red warning. It's very important to get the user's attention here.